docs/release/api/insertable-table.md

```diff
@@ -1,5 +1,10 @@
 # InsertableTable
 
+Import conventions:
+```python
+import pixeltable as pxt
```

Comment:
Just to remind ourselves: did we conclusively decide on pxt or is px still in the running?